static_src/components/elastic_line_item.jsx

```diff
@@ -20,18 +17,12 @@ const defaultProps = {
 };
 
 export default class ElasticLineItem extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.styler = createStyler(style);
-  }
-
   render() {
     const props = this.props;
```

Comment:
Not really related, but if you're here, might want to remove `props` tmp var - not really conventional to pull out `this.props` into a var.